log/zap.go

```diff
@@ -158,6 +158,11 @@ func Array(name string, array ArrayMarshaler) Field {
 	return Field(zap.Array(name, array))
 }
 
+// Strings for logging slice of strings.
+func Strings(name string, slice []string) Field {
```

Comment:
don't want to use ArrayMarshaler?

Reply:
Idk what you mean. `[]string` doesn't implement `ArrayMarshaler`.

Reply:
instead of defining a function for every type of array you want to log, you can do something like this

https://github.com/spacemeshos/go-spacemesh/blob/a316d780f95241a38cc0db1cf3a0645ef0891811/miner/oracle.go#L175

the advantage is that the marshaling is lazy.

Reply:
but this doesnt really matter. feel free to leave the code as is.